src/Renci.SshNet/Security/Cryptography/Ciphers/CipherPadding.cs

```diff
@@ -54,5 +54,12 @@ public byte[] Pad(byte[] input, int paddinglength)
         /// The padded data array.
         /// </returns>
         public abstract byte[] Pad(byte[] input, int offset, int length, int paddinglength);
+
+        /// <summary>
+        /// Gets the padd count from the specified input.
+        /// </summary>
+        /// <param name="input">The input.</param>
+        /// <returns>The padd count.</returns>
+        public abstract int PadCount(byte[] input);
```

Comment:
We can consider replacing `CipherPadding` and `PKCS7Padding` with BouncyCastle's `Pkcs7Padding`

Reply:
Let me have a try

Reply:
Just had a quick look. It would be easier to remove it in my next PR https://github.com/sshnet/SSH.NET/pull/1546.